bin/blin.p6

```diff
@@ -58,17 +58,40 @@ my $skip-tests = (
    ‘Time::Duration’,
    ‘Toaster’,
    ‘Uzu’,
+   # Do not check modules that are broken and seemingly abandoned.
+   # we tried to reach out to authors, but no response for months leaves
+   # these in a limbo we don't want to spend time on
+   'Acme::Flutterby', 'Acme::Mangle', 'Acme::Skynet',
```

Comment:
I'm not sure about this. The output produced by blin is used from time to time to populate https://github.com/perl6/ecosystem-unbitrot (which creates/closes tickets as needed). What's the disadvantage in testing modules that are known to always fail, unless they're flappers?

Reply:
These modules were abandoned by authors for different reasons.
We did send issues, we did write patches and PR them. Then months and months passed and no action was taken.
They add clutter to release process. You have to check failures when cutting a release, and you have a big bunch of modules which are just broken. You don't remember each one of them and for each one you are checking if the author is aware, if there is an issue, "Oh, I saw that already a half of year ago, nothing has changed, just a false positive I spent time on".
When there are 10 broken modules that's ok, but if you have more, it is not so easy anymore.
I don't see any merit in creating new tickets from this list to ecosystem-unbitrot repo. It was there already. It didn't work out after all. The disadvantage is time spent. The advantage is... I don't see one, really.